byway/src/main/webapp/style.css

```diff
@@ -9,25 +9,25 @@
 }
 
 .btn {
-    margin-left: 100px;
-    margin: 10px;
-    align-items: center;
-    background-color: #DDDDDD;
-    border-color: #FFFFFF;
-    border-radius: 10px;
-    border-width: 1px;
-    padding: 10px;
-    outline: none;
+  margin-left: 100px;
+  margin: 10px;
```

Comment:
consider using condensed version of margin (margin: 10px 100px 10px 10px, in this case)

Reply:
Fixed